api/MediaDevices.json

```diff
@@ -585,6 +1125,42 @@
               "deprecated": false
             }
           }
+        },
+        "constraints_resizeMode_parameter": {
```

Comment:
@caugner I originally did this under  `MediaDevices.getUserMedia()` because the documentation work I'm looking at is for  `resizeMode` on that method. 

So I'd quite like to record at least this particular constraint on this method. I've done that here but its a TBD because I am _assuming_ that the other browsers match the data in the `getUserConstraints` method.
Or do you think I should make that assumption and duplicate everything here, or do something else.

Note, also pending your response on whether you want this all under a sub feature for the constraint parameter.

Reply:
Thought about it shortly, and I think it would be fine to have this also directly under `getUserMedia()`, e.g. `accepts_aspectRatio_constraint`.

Would be interesting to hear what @Elchi3 thinks about this?

Reply:
> I believe `constraints_resizeMode_parameter` is the schema form (not _accepts_ prefix). Though I am just as happy with a `constraints_parameter` feature that has subfeature `resizeMode_property`

I think this is closest to what we have in https://github.com/mdn/browser-compat-data/tree/main/docs/data-guidelines#parameters-and-parameter-object-features so I would vote to go in that direction. 

(I think somewhere we should have guidelines or maybe even a tool, where we will be able to input IDL structures and output BCD structures, so that we don't run into structural questions all the time and also make BCD more consistent overall)

Reply:
@caugner So based on ^^^ this method matches the expected schema.

Note that this just duplicates the support data of `getSupportedConstraints()` since that is what we can test. I think that is reasonable.

I have updated `getSupportedConstraints()` to use the same pattern as there are other cases that do so in BCD already- see https://github.com/mdn/browser-compat-data/pull/27891/files#r2409028614